sp800-63a/sec6_dc.md

```diff
@@ -15,5 +15,5 @@ There are two specific use cases for deriving identity:
 1. A _claimant_ seeks to obtain a derived PIV, bound to their identity record, for use only within the limits and authorizations of having a PIV smartcard. *This use case is covered in SP 800-157 Guidelines for Derived Personal Identity Verification (PIV) Credentials, [[SP 800-157]](#SP800-157)*. 
 2. An _applicant_ seeks to obtain an authenticator, bound to a proofed identity record, from a CSP with which the individual does not have a pre-existing relationship. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g. basic browsing vs. financial). *This use case is covered by allowable identity evidence in [Section 5.2](#validate).*
 
-As stated above, all requirements for PIV derived credentials can be found in [SP 800-157](#SP800-157).  For the second use case described above, this guideline does not differentiate between physical and digital identity evidence. Nor does it consider the authenticator issued as a result of providing digital identity evidence as special. Therefore it is acceptable, based on the characteristics of the authenticator, and potentially an assertion generated by the primary CSP, to be used at identity evidence for IAL2 and 3. In addition, any authenticator issued based on digital identity evidence is subject to the requirements of [SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html).
+As stated above, all requirements for PIV derived credentials can be found in [SP 800-157](#SP800-157).  For the second use case described above, this guideline does not differentiate between physical and digital identity evidence. Therefore it is acceptable, if the authenticator or  an assertion generated by the primary CSP meet the requirements of [Section 5](#sec5), to be used at identity evidence for IAL2 and 3. In addition, any authenticators issued as a result of providing digital identity evidence is subject to the requirements of [SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html).
```

Comment:
"to be used" -> "for them to be used"
"is subject" -> "are subject"
SP800-63B -> SP 800-63B